### ISSUE TITLE
Issue with stage three training 

### ISSUE THREAD

**Issue by liuxin99** [StuartMesham/gector_experiment_public#3]
I have been trying to replicate your training results for the third stage. However, I'm facing an issue where the validation loss only decreases during the first 1-2 epochs and then keeps increasing. I would like to know how many epochs you trained for before stopping. It would be really helpful if you could provide some insights or guidance on reproducing your results.

**Comment by StuartMesham** [comment]
Hi @liuxin99, that sounds consistent with what we saw. We used early stopping with a patience of 3 evals (and use the best scoring checkpoint, rather than the last one), and for phase 3 the first or second eval was often the best.

To help with reproducing our results, here are the logs for [phase 1](https://wandb.ai/stuartmesham/lemon-spell_5k/runs/snhhc89q), [phase 2](https://wandb.ai/stuartmesham/lemon-spell_5k/runs/mm3y1vvv) and [phase 3](https://wandb.ai/stuartmesham/lemon-spell_5k/runs/i2nho5xe) of the training for the `deberta-large_lemon-spell_5k_4_p3` model.

**Comment by liuxin99** [comment]
@StuartMesham Thank you very much!

**Comment by liuxin99** [comment]
@StuartMesham  Sorry, can you provide me the logs for the `deberta-large_basetages_10k_1_p3` model?

**Comment by StuartMesham** [comment]
@liuxin99 Sure! Here are [phase 1](https://wandb.ai/stuartmesham/basetags_10k/runs/yxjtmv01), [phase 2](https://wandb.ai/stuartmesham/basetags_10k/runs/441ta301), and [phase 3](https://wandb.ai/stuartmesham/basetags_10k/runs/rjc58k05) logs for `deberta-large_basetags_10k_1_p3`.
